cryptoprotocol/src/main/java/de/upb/crypto/incentive/cryptoprotocol/model/PublicParameters.java

```diff
@@ -0,0 +1,24 @@
+package de.upb.crypto.incentive.cryptoprotocol.model;
+
+
+import de.upb.crypto.craco.prf.aes.AesPseudorandomFunction;
+import de.upb.crypto.craco.sig.sps.eq.SPSEQSignatureScheme;
+import de.upb.crypto.math.interfaces.structures.GroupElement;
+import de.upb.crypto.math.pairings.generic.BilinearGroup;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * a class representing the public parameters of the 2020 incentive system
+ */
+
+@Data
+@AllArgsConstructor
+public class PublicParameters
+{
+    private BilinearGroup bg;
+    private GroupElement w;
+    private GroupElement h7;
+    private AesPseudorandomFunction prf; // not in paper, but we need to store PRF that is used in incentive system instance somewhere
```

Comment:
Do you really need to store this?

Reply:
I will note though (not saying this should be addressed in this pull request) that you'll later probably want to replace the `AesPseudorandomFunction` with something that will help you more to generate vectors of exponents. But we'll cross that bridge when we come to it.

Reply:
Good point.